tests/datasets/configuration/test_entity_loader.py

```diff
@@ -12,44 +12,65 @@
 )
 from snuba.datasets.configuration.entity_builder import build_entity_from_config
 from snuba.datasets.entities.entity_key import EntityKey
-from snuba.datasets.entities.factory import get_entity
+from snuba.datasets.entity import Entity
+from snuba.datasets.factory import reset_dataset_factory
 from snuba.datasets.pluggable_entity import PluggableEntity
 from snuba.query.expressions import Column, FunctionCall, Literal
 
+reset_dataset_factory()
```

Comment:
This will conflict with the testing changes I rolled out. If you move your tests inside the `ConfigurationTest` class you shouldn't need this call.